external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -3,104 +3,125 @@
 
 import * as fs from 'fs';
 import { FRAME_LIFETIME, ModuleInfo } from './utils';
+import { IRuntimeCompundValue, RuntimeValueType } from './runtime';
+
 
 // Data types corresponding to trace file JSON schema.
 
-interface ITraceModule {
+interface JSONTraceModule {
```

Comment:
These renamings are unrelated to the main purpose of this PR but things were getting too confusing with the old naming scheme